src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,25 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class MainApp {
+    public static void main(String[] args) {
+
+        Truck truck = new Truck("Truck");
+        Bulldozer bulldozer = new Bulldozer("Bulldozer");
+        Excavator excavator = new Excavator(" Excavator");
```

Comment:
Instead of using specific class types for the references `truck`, `bulldozer`, and `excavator`, use the abstract class type `Machine`. This will allow you to take advantage of polymorphism. For example:
Machine truck = new Truck("Truck");
Machine bulldozer = new Bulldozer("Bulldozer");
Machine excavator = new Excavator("Excavator");